source/spp/mnm/spp_117_nl_abiotiek_waterlopen/05_kwaliteitszorg.Rmd

```diff
@@ -0,0 +1,30 @@
+# Extra aspecten van kwaliteitszorg t.o.v. de subprotocols
+
+Voor bepaalde werkzaamheden gelden er mogelijk specifieke richtlijnen rond kwaliteitszorg.
+Hiervoor kan best het hoofdstuk zie kwaliteitszorg van deelprotocollen ([sfp-113-nl 2023.04](../2023.04/index.html), [sfp-114-nl 2024.03](../2024.03/index.html), [sfp-115-nl 2024.04](../2024.04/index.html)) worden geraadpleegd.
+
+## Richtlijnen
+
+-   Vul het staalnameformulier steeds correct, nauwkeurig, eenduidig en duidelijk leesbaar in met potlood of onuitwisbare, watervaste inkt.
+    Zorg ervoor dat het staalnameformulier niet kan wegwaaien, neem eventueel regelmatig een foto van het formulier uit voorzorg.
+
+-   Neem steeds alle relevante protocollen mee in het veld en kijk voor het vertrek steeds na of alle materiaal aanwezig is en op batterijen werkende toestellen voldoende opgeladen zijn.
+
+-   Volg de protocollen strikt op om niet af te wijken van de methodiek en de correctheid van de resultaten te waarborgen.
+
+-   Klasseer het staalnameformulier achteraf correct, zodat deze, ook na lange tijd, gemakkelijk te raadplegen zijn.
+
+## Vereiste competenties
+
+Enkele vereiste competenties om kwaliteitsvolle gegevens te bekomen:
+
+-   Nauwkeurigheid: de uitvoering van veldmetingen en het vastleggen van veldwaarnemingen vereisen een grote mate van accuratesse.
+
+-   Vermogen te plannen en te organiseren: bij de uitvoering van meetprogramma’s moeten tal van werkzaamheden gepland, georganiseerd en op elkaar afgestemd worden.
+    De staalnemer moet in staat zijn om hier zelfstandig of in overleg met de projectleider uitvoering aan te geven.
+
+-   Zelfstandigheid: de staalnemer moet in staat zijn om het merendeel van de werkzaamheden zelfstandig (op locatie) uit te voeren.
```

Comment:
en ook zelfstandig in staat kunnen zijn om de veiligheid van de situatie ter plaatse in te schatten.

Reply:
Heb ik nu toegevoegd

Reply:
dat kan je in feite ook reken onder werkzaamheden; eventueel als volgt te formuleren: de staalnemer moet in staat zijn om de werkzaamheden zelfstandig uit te voeren en in staat kunnen zijn om de veiligheid van de situatie ter plaatse in te schatten.